### ISSUE TITLE
Google Search App (GSA) on iOS not detected

### ISSUE THREAD

**Issue by connyhald** [faisalman/ua-parser-js#175]
We often get requets to our web app coming from the Google Search App on iOS. It would be great if ua-parser-js could detect it as well. Examples below.

```
Mozilla/5.0 (iPad; CPU OS 9_3 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) GSA/14.0.119004557 Mobile/13E237 Safari/600.1.4

Mozilla/5.0 (iPad; CPU OS 9_2 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) GSA/11.1.66360 Mobile/13C75 Safari/600.1.4

Mozilla/5.0 (iPhone; CPU iPhone OS 9_2_1 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) GSA/11.1.66360 Mobile/13D15 Safari/600.1.4

Mozilla/5.0 (iPhone; CPU iPhone OS 8_4_1 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) GSA/11.1.66360 Mobile/12H321 Safari/600.1.4
```

**Comment by ebbmo** [comment]
Hi @connyhald,

Good point. Do you mind to open a PR for this issue (including tests)? 

KR, ebbmo

**Comment by brophdawg11** [comment]
This is also an issue for us - opened https://github.com/faisalman/ua-parser-js/pull/265 to address it.

**Comment by faisalman** [comment]
Fixed in #265, thanks @brophdawg11
